src/components/Calculator.js

```diff
@@ -1,77 +1,41 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import { useState } from 'react';
+import Row from './Row';
 import './Calculator.css';
 
-const Button = (props) => {
-  const { buttonName } = props;
-  return (
-    <button type="button">
-      {buttonName}
-    </button>
-  );
-};
-
-Button.propTypes = {
-  buttonName: PropTypes.string.isRequired,
-};
-
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { value: 0 };
-  }
-
-  render() {
-    const { value } = this.state;
-    const buttons = [
-      ['AC', '+/-', '%', '/'],
-      ['7', '8', '9', 'X'],
-      ['4', '5', '6', '-'],
-      ['1', '2', '3', '+'],
-    ];
-    return (
-      <div id="calculator">
-        <div id="display-calc">
-          {value}
-        </div>
+import calculate from '../logic/calculate';
 
-        <div id="calc-buttons">
-          <div className="row">
-            {buttons[0].map((name) => (
-              <Button key={name} buttonName={name} />
-            ))}
-          </div>
+const Calculator = () => {
+  const [calcObj, setCalcObj] = useState({});
+  const { next, total } = calcObj;
 
-          <div className="row">
-            {buttons[1].map((name) => (
-              <Button key={name} buttonName={name} />
-            ))}
-          </div>
+  const buttons = [
+    ['AC', '+/-', '%', '/'],
```

Comment:
- [ ] <b>REQUIRED:</b> Division is not working, that because you are using this sign `/` as an operator when in operate.js, there are expecting this one `÷` as operator. Kindly fix it.

Reply:
Fixed division